### ISSUE TITLE
Problem with signing of PFSFeeUpdate messages

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#4343]
During the scenario runs I see some warning like the following. Seems like something is wrong with the signature handling.

```
pfs-goerli_1            | 2019-07-05 09:10:37.872052 [warning  ] Message not signed by sender!  [raiden_libs.matrix] message={'signature': '0xfd4096d147b74f7b28a3e3c556a3bead616fb1f05b3789ac1b797a133c347e382751c3ea0d4451f985329271c7bc5d02d2b4ddf9d976d8bead7c5ca8e8faa40d1b', 'canonical_identifier': {'chain_identifier': 5, 'token_network_address': b'\xbd\xce\xc1\x17\xd4\x11\x8d=\xf3S\xef%\\\xd9|\x90A\xcd\x95\xbe', 'channel_identifier': 1}, 'updating_participant': '0x00E5473fC25aa811c11b4Ce310Ca91f4eA94f168', 'fee_schedule': {'flat': 0, 'proportional': 0, 'imbalance_penalty': None, '_penalty_func': None}, 'timestamp': datetime.datetime(2019, 7, 5, 9, 10, 36), 'event_name': 'PFSFeeUpdate'} peer_address=0x00E5473fC25aa811c11b4Ce310Ca91f4eA94f168 signer=0x790d21105FC8D3591465821E1A7537b15622712b
```

**Comment by karlb** [comment]
Marshmallow serialized datetimes differently, depending on whether `datetutil` is installed or not.

With dateutil: `2019-07-08T10:51:05.559672+00:00`
Without dateutil: `2019-07-08T10:51:05+00:00`

Raiden and services use the same serialization code to generate the packed data for the signature, but raiden has `dateutil` installed while the services don't. This leads to different signatures despite using the same code path.

**Comment by karlb** [comment]
From the changelog for marshmallow 3.0.0rc8 which has just been released four days ago:
> python-dateutil is no longer used. This resolves the inconsistent behavior based on the presence of python-dateutil (#497, #1234).

Looks like updating marshmallow will solve that problem.

**Comment by palango** [comment]
PR for that is here: https://github.com/raiden-network/raiden/pull/4334

I just couldn't get the build to complete yet, mostly because of https://github.com/raiden-network/raiden/issues/4340

**Comment by palango** [comment]
Hopefully fixed by #4334
